UI/org.eclipse.birt.report.designer.ui.editors/build.properties

```diff
@@ -1,5 +1,5 @@
 source.. = src/
-output.. = bin/
```

Comment:
I'm not sure if this change may cause eclipse PDE's problem. Did you test PDE environment in eclipse IDE?
BIRT source code not only support maven, but also support traditional plugin PDE. Using different output folder can make sure PDE and maven work simultaneously.

Reply:
This change is motivated by Eclipse PDE and fixes a warning inside Eclipse.

To get a buildable PDE workspace, I imported the BIRT projects using maven import. In this process the m2e connector updates the Eclipse projects using the meta data in the pom.xml. This makes "target/classes" the output folder of the projects. This in turn leads to a warning from PDE along the lines of "'bin/' is not an output folder of the project".